src/helpers/redisToken.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from 'dotenv'
+import { createClient } from 'redis'
+
+dotenv.config()
+
+const redisClient = createClient({
+  password: process.env.REDIS_PASSWORD,
+  socket: {
+    host: process.env.REDIS_HOST,
+    port: parseInt(process.env.REDIS_PORT, 10),
```

Comment:
```suggestion
  password:.REDIS_PASSWORD,
  socket: {
    host: REDIS_HOST,
    port: REDIS_PORT,
```
We can export and  import this from the `src/config`